examples/plot_snia_curve_fits.rs

```diff
@@ -136,8 +137,18 @@ fn main() {
             )
             .into(),
         ));
+        features.push((
+            "LinexpFit Ceres",
+            LinexpFit::new(
+                CeresCurveFit::default().into(),
```

Comment:
We need to move it above, inside `#[cfg(all(feature = "gsl", feature = "ceres-source"))] {}`